test/NNODE_tests.jl

```diff
@@ -204,3 +204,29 @@ sol = solve(prob, NeuralPDE.NNODE(luxchain, opt; batch = true), verbose = true,
             maxiters = 400,
             abstol = 1.0f-8, dt = 1 / 5.0f0)
 @test sol.errors[:l2] < 0.5
+
+using OrdinaryDiffEq, DiffEqFlux, OptimizationPolyalgorithms, Lux, Random, Flux, OptimizationOptimJL, Test, Statistics
+
+function f(u, p, t)
+    [p[1] * u[1] - p[2] * u[1] * u[2], -p[3] * u[2] + p[4] * u[1] * u[2]]
+end
+
+p = [1.5, 1.0, 3.0, 1.0]
+u0 = [1.0, 1.0]
+prob_oop = ODEProblem{false}(f, u0, (0.0, 3.0), p)
+true_sol = solve(prob_oop, Tsit5(), saveat = 0.01)
+func = Lux.σ
+N = 12
+chain = Lux.Chain(Lux.Dense(1, N, func), Lux.Dense(N, N, func), Lux.Dense(N, N, func),
+                    Lux.Dense(N, N, func), Lux.Dense(N, length(u0)))
+
+opt = ADAM(0.01)
+dx = 0.05
+weights = [0.9, 0.09, 0.009]
+samples = 10000
+alg = NeuralPDE.NNODE(chain, opt, autodiff = false, strategy = NeuralPDE.WeightedGridTraining(dx, weights, samples))
+sol = solve(prob_oop, alg, verbose=true, maxiters = 100000, saveat = 0.01)
+
+println(abs(mean(true_sol .- sol)))
+println(abs(mean(sol) - mean(true_sol)))
+@test abs(mean(sol) - mean(true_sol)) < 0.5
```

Comment:
Make a Lotkva-Volterra test case?

Reply:
I see, a bit oddly written but it works. mean difference of 0.5: isn't that quite high? The pictures were a bit better than that?

Reply:
Yep, I only used 0.5 because that is what was used in the other tests. The value is always much lower than that, I can make it test threshold lower if you want me to